lib/reckon/app.rb

```diff
@@ -151,6 +167,14 @@ def output(ledger_line)
       options[:output_file].flush
     end
 
+    def most_specific_regexp_match( row )
+      matches = []
+      regexps.keys.sort_by { |s| -s.inspect.size }.each do |regexp_key|
```

Comment:
I'm not sure what's the best way. My intention was to make the most specific regexps first, so that /webhosting/ would be higher priority than /host/. Both regexp size and match size become non-intuitive in more complex regexps.